setup.py

```diff
@@ -13,6 +14,12 @@
     ]
 )
 
+package_data = {"attrs_strict": ["py.typed"]}
+for root, dirs, files in os.walk("attrs_strict"):
```

Comment:
A better approach to add these via a recursive include https://docs.python.org/3/distutils/sourcedist.html